File: 프로그래머스/unrated/181928. 이어 붙인 수/이어 붙인 수.py
def solution(num_list):
    even = []
    odd = []
    for num in num_list:
        if num % 2 == 0:
            even.append(num)
        else:
            odd.append(num)
    answer = int(''.join(map(str, even))) + int(''.join(map(str, odd)))
    return answer

File: 프로그래머스/0/181928. 이어 붙인 수/이어 붙인 수.java
import java.lang.StringBuilder;

class Solution {
    public int solution(int[] num_list) {
        StringBuilder sb_odd = new StringBuilder();
        StringBuilder sb_even = new StringBuilder();
        for (int num: num_list) {
            if (num % 2 == 0) {
                sb_even.append(num);
            } else {
                sb_odd.append(num);
            }
        }
        return Integer.parseInt(sb_odd.toString()) + Integer.parseInt(sb_even.toString());
    }
}